### DISCUSSION TITLE
一则惊天消息:传奇绳匠法厄同账号注销

### DISCUSSION THREAD

**Discussion by libones** [share121/inter-knot#4914] [其他]
传奇绳匠法厄同账号注销，一夜之间如人间蒸发，目前为止还没有任何消息，疑似被捕

**Comment by shidijun** [comment]
不该吧 人家毕竟是传奇绳匠 总不能连治安局都防不住吧

**Comment by Cypher2024** [comment]
完口拉，法厄同大佬也落网了

**Comment by Minapocalis** [comment]
不！！！法厄同我的偶像！他一定没有落网
![img_-375810294_1707339301538](https://github.com/user-attachments/assets/a890d335-7a77-4aee-9a82-7f5d1772d96d)
